### PR TITLE
Move queuing logic for scaling from ui-classic

### PR DESCRIPTION
**Pull request by rwsu**
Moved logic into this repo for deciding whether to use Mistral
workflows if they are available or whether to use direct stack
update if the undercloud environment is OSP 9 or lower.

### CODE REVIEW

**Comment by tzumainn** [comment]
Is there a corresponding UI PR to make sure that the method changes are in sync, or is this completely standalone?

**Comment by tzumainn** [comment]
Oh, nevermind, I see it now

File: app/models/manageiq/providers/openstack/infra_manager/orchestration_stack.rb [hunk 1]
```diff
@@ -138,5 +160,17 @@ def log_and_raise_update_error(method_name, err)
     raise MiqException::MiqOrchestrationUpdateError, err.to_s, err.backtrace
   end
 
+  def workflow_service
+    ext_management_system.workflow_service
```
Comment (mansam) [inline app/models/manageiq/providers/openstack/infra_manager/orchestration_stack.rb]: Should this be `ext_management_system.openstack_handle.workflow_service` or (or `detect_workflow_service` to handle the exception)?

Comment (mansam) [reply app/models/manageiq/providers/openstack/infra_manager/orchestration_stack.rb]: Nevermind, I see that `ext_management_system.workflow_service` is actually calling `ext_management_system.openstack_handle.detect_workflow_service` on the InfraManager.

**Pull request merged**